www/nguv03/cv09/facebook/profile.php

```diff
@@ -11,7 +11,7 @@
 
 $fb = new \Facebook\Facebook(array_merge(CONFIG_FACEBOOK, ['default_access_token' => $_SESSION['fb_access_token']]));
```

Comment:
tahle zmena byla asi udelana omylem - prosim odstrante

Reply:
opraveno, prepisoval jsem nazev promenne a Phpstorm to prepsal pro cele repo...

Reply:
opraveno, prepisoval jsem nazev promenne a Phpstorm to prepsal pro cele repo...